### ISSUE TITLE
ENOENT error

### ISSUE THREAD

**Issue by tlswoo** [cadon0/gif-to-the-beat#39]
This error seems to have occurred previously at #36 ; here's what mine says:

```Disconnected from feed of osu! data, reconnecting...
Connected to feed of osu! data. Data will be logged occasionally to show it's active
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[Error: ENOENT: no such file or directory, open 'C:\Θä£╠úΘìÇ╠ú'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Θä£╠úΘìÇ╠ú'
}
```

If you need any other info for debugging, let me know!

**Comment by cadon0** [comment]
Thanks! Fixed today, see #38 (yes you're correct, same as #36)

Solution:
> Please download the new [GifToTheBeatProvider.exe](https://github.com/cadon0/ProcessMemoryDataFinder/releases) and replace the old one ([direct download link](https://github.com/cadon0/ProcessMemoryDataFinder/releases/download/v1.0.3/GifToTheBeatDataProvider.exe))

**Comment by tlswoo** [comment]
Thanks for the fix!
